pyccel/commands/lambdify.py

```diff
@@ -0,0 +1,89 @@
+"""
+File describing commands associated with the lambdify function which converts a
+SymPy expression into a Pyccel-accelerated function.
+"""
+import sympy as sp
+from packaging.version import parse
+
+from pyccel.epyccel import epyccel
+from pyccel.utilities.strings import random_string
+
+if parse(sp.__version__) >= parse('1.8'):
+    from sympy.printing.numpy import NumPyPrinter
+else:
+    from sympy.printing.pycode import NumPyPrinter
```

Comment:
May I suggest to avoid the standalone "parse" function? We already have enough parsers in Pyccel 😉 
For example:

```suggestion
import sympy as sp
from packaging import version

from pyccel.epyccel import epyccel
from pyccel.utilities.strings import random_string

if version.parse(sp.__version__) >= version.parse('1.8'):
    from sympy.printing.numpy import NumPyPrinter
else:
    from sympy.printing.pycode import NumPyPrinter
```